### DISCUSSION TITLE
Does imgproxy allow multiple gradients to be added?

### DISCUSSION THREAD

**Discussion by martinarvaiaccedo** [imgproxy/imgproxy#1397] [Q&A]
Is there a possibility to add multiple gradients? To sort of achieve something like this.
Notice there is a gradient from bottom to top and from bottom left to middle.

I suspect this is possible to achieve with full size watermarks but a better solution would be to be able to add multiple gradient parameters. 

![image](https://github.com/user-attachments/assets/f840bb6e-79eb-4534-bf8f-998fafd29396)

Thank you for your answer!

**Comment by DarthSim** [comment]
Hey @martinarvaiaccedo!

Yes, you can apply multiple gradients using chained pipelines: `.../gr:.../-/gr:.../...`

**Reply by martinarvaiaccedo** [reply]
@DarthSim , thanks so much for the quick response. Now I found it in the docs as you mention pipelines.  https://docs.imgproxy.net/features/chained_pipelines   Perhaps a quick reference to places where you can chain things would be a good idea.  Anhow, I'm happy pro supports this feature!
